online-shop/src/components/listProducts/listProducts.component.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import Product, {Produs} from "../product/product.component"
+
+import "./listProducts.styles.css"
+
+
+
+const products_list: Array<Produs>= 
+[ {id:1,name:"tv", category:"electronics", price:100, imageUrl:"https://images.unsplash.com/photo-1579894059380-1866b68bce6b?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1350&q=80"},
```

Comment:
As far as I can see from the indentation and formatting you haven't used either the editor formatter or Prettier. I recommend you to use one of them because it will beautify the code and bring it to a formatting standard